keyboards/hazel/chaz/rules.mk

```diff
@@ -0,0 +1 @@
+CAPS_WORD_ENABLE = yes
```

Comment:
We generally dont want this enabled by default (nor mixing rules.mk/info.json enabling of features)
```suggestion
# This file intentionally left blank
```

Reply:
changes commited, and I also removed `DOUBLE_TAP_SHIFT_TURNS_ON_CAPS_WORD` from default `config.h`